src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: IBM.com Style Models
+description: IBM.com Style Models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+tabs: ['Overview', 'Style Model List']
+---
+<PageDescription>
+
+IBM.com Style Models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Our vision</AnchorLink>
+<AnchorLink>Style Model</AnchorLink>
+<AnchorLink>Using Style Models</AnchorLink>
+
+</AnchorLinks>
+
+
+## Our vision
+
+Our vision is that every visitor to IBM.com has a productive, efficient and delightful experience.
+
+
+IBM.com is a collection of highly functional, versatile, and diverse contents and experiences. It serves different audience groups and their needs, in different industries and geographic regions, and covers crucial portion of our users' journeys.
```

Comment:
```suggestion
IBM.com is a collection of highly functional, versatile, and diverse contents and experiences. It serves different audience groups and their needs, in different industries and geographic regions, and covers a crucial portion of our users' journeys.
```

Reply:
This suggestion was added separately. Thank you!